### DISCUSSION TITLE
qingping/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#13] [Ideas]
# qingping/

这是一个非常简单的分享。 和 青萍空气检测仪 Lite简单拆解 类似，尝试拆开青萍空气检测仪 Lite，不过我的目的是尝试解决它风扇异响的问题（间隔性

https://nova.moe/qingping/

**Comment by AnLuoRidge** [comment]
谢谢分享，让我躲过一次徒劳的风扇维修。那我试试能不能淘宝买个同类型的传感器风扇。

**Comment by zhendery** [comment]
我真服啦，我也是风扇异响想拆开上油，结果这风扇是一体的根本没法拆开上油，直接给掰断了。然后温湿度传感器的排线也在多次拆装测试下暴毙，飞线飞了半天愣是飞不上排线越剪越短，最终不得不放弃现在也是没有温湿度读数。这个传感器某宝有同款，但是焊接好排线的就是找不到。
